package.json

```diff
@@ -3,15 +3,20 @@
   "version": "2.0.1",
   "description": "Core contracts for Aragon",
   "scripts": {
-    "test": "npm run ganache-cli:dev && truffle test --network rpc",
```

Comment:
Why are these renamed back to testrpc? I intentionally renamed them after their rename when updating the dependencies on #169

Reply:
I think I now understand the rationale behind it. IMO we should rename the script and references to something other than `testrpc` as it is easy to think only the testrpc client is run. My suggestions would be `test-nodes.sh`, `start-nodes.sh` or `manage-nodes.sh`

Reply:
I agree. Wasn't quite sure what to name that back to. Seems like the same file name should be applied in the aragon apps as to avoid confusion there as well.

Reply:
I agree